src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/helper/EntityDataLoader.kt

```diff
@@ -29,7 +29,7 @@ class VisitEntityHelper(
     prisonerId: String = "FF0000AA",
     prisonId: String = "MDI",
     visitRoom: String = "A1",
-    visitStart: LocalDateTime = LocalDateTime.of(2021, 11, 1, 12, 30, 44),
+    visitStart: LocalDateTime = LocalDateTime.of((LocalDateTime.now().year + 1), 11, 1, 12, 30, 44),
```

Comment:
I think you could change this to LocalDateTime.now(),...

Realised the testing was on  telemetry assert, I think it's because it uses ISO_DATE_TIME so instead of truncating it I would advise the following or something similar :

assertThat(it["visitStart"]).isEqualTo(reservedVisit.visitStart.format(DateTimeFormatter.ISO_DATE_TIME))

What ever the case I think the above assert is better way of testing this code

Reply:
Updated the asserts for visitStart sent to telemetry client as above.